bazel/envoy_build_system.bzl

```diff
@@ -382,3 +415,10 @@ def envoy_select_exported_symbols(xs):
         "@envoy//bazel:enable_exported_symbols": xs,
         "//conditions:default": [],
     })
+
+def envoy_select_force_libcpp(xs, ys = None):
```

Comment:
nitpicking: These param names are kinda opaque, can you use something like `if_libcpp`?